Sources/Build/misc.swift

```diff
@@ -32,6 +35,83 @@ extension CModule {
     }
 }
 
+extension ClangModule {
+    
+    public enum ModuleMapError: ErrorProtocol {
+        case UnsupportedIncludeLayoutForModule(String)
+    }
+    
+    public func generateModuleMap() throws {
+        
+        //Return if module map is already present
+        guard !moduleMapPath.isFile else {
```

Comment:
Indeed, I did not realize we were not generating them to the build directory. We should be doing that if we are going to generate *at all*.